### DISCUSSION TITLE
Cycling F12 key to open and close bookmarks not working

### DISCUSSION THREAD

**Discussion by 13R4F431** [sumatrapdfreader/sumatrapdf#2603] [General]
Hi, I would like to talk about an issue. I generally used the F12 key to cycle open and close the bookmarks tab, usually at previous versions worked flawlessly. But at the version 3.4.14.14509 64bit when the bookmarks windows opens, the focus goes to bookmarks window, and the F12 key don't work to close the bookmarks. I have to click at the text window to allow F12 key to work again, closing the bookmarks window. Can I do something for the F12 key to came back to cycle open and close the bookmarks?

**Comment by kjk** [comment]
Sorry about that. I've already fixed it.

If I make 3.4.2 release, the fix will be in it.

It'll also be in 3.5 pre-release. I've stopped making 3.5 pre-releases while I'm monitoring 3.4 release rollout, but I'll restart them in a couple of days.

**Reply by 13R4F431** [reply]
I would like to thank you for your attention and for fixing the issue.
